### DISCUSSION TITLE
Update to Vue3 slot syntax

### DISCUSSION THREAD

**Discussion by seggewiss** [shopware/platform#3164] [RFC / Proposal]
**Effort**: low
**Priority**: high

### Description: 
Taking the first important step to a Vue3 update, we will deprecate and upgrade the Vue2 slot syntax.
The following `eslint` rules will be added to prevent the old syntax from being used:

- `'vue/v-slot-style': ['error'],`
- `'vue/no-deprecated-slot-attribute': ['error'],`
- `'vue/no-deprecated-slot-scope-attribute': ['error'],`

### Benefits
This will allow us to update to Vue3 more smoothly with the 6.6 Major release.

### Break strategy
This will be broken after the last minor release of the 6.4 series and published with the 6.5 major.
As implementing this via feature flags is inconvenient and the benefit is not worth the effort.

**Comment by lasomethingsomething** [comment]
Due to lack of activity around this in recent years and the likelihood that this won't change anytime soon, we'll close this proposal for now. cc @keulinho

**Reply by mitelg** [reply]
it is more like, that this is already done, as we are not on VueJs 3
